website/cue/reference/components/sinks/base/nats.cue

```diff
@@ -383,6 +383,15 @@ base: components: sinks: nats: configuration: {
 			}
 		}
 	}
+	jetstream: {
+		description: """
```

Comment:
```suggestion
		description: 
```
actually these should be removed

Reply:
This file has been generated from the doc comments added in the code, so it won't be possible to remove these quotes. They are a part of `cue` and don't make it all the way through to the final docs on the vector.dev website..